plugin-server/src/cdp/hog-transformations/hog-transformer.service.ts

```diff
@@ -164,20 +168,38 @@ export class HogTransformerService {
         return promises
     }
 
-    public transformEvent(event: PluginEvent): Promise<TransformationResult> {
+    public transformEventAndProduceMessages(event: PluginEvent): Promise<TransformationResult> {
```

Comment:
Wasn't sure of the naming here but basically wanted one function that does the transform and another that does it _and_ produces the kafka messages